src/util/LoggerUtil.ts

```diff
@@ -0,0 +1,9 @@
+export class LoggerUtil {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	public static shortenJiraError( error: any ): string {
+		return `Information:
+		  Status code: ${ error.response?.status || undefined }
+		  Status text: ${ error.response?.statusText || undefined }
+		  Error message(s): ${ error.response?.data.errorMessages || undefined }`.replace( /\t/g, '' );
```

Comment:
Potential NPE could occur here. I also don't see the need for `|| undefined` as it could replace 0 or empty strings with `undefined`.

```suggestion
		  Status code: ${ error?.response?.status }
		  Status text: ${ error?.response?.statusText }
		  Error message(s): ${ error?.response?.data?.errorMessages }`.replace( /\t/g, '' );
```

Reply:
Thanks! Sorry, I'm not sure how I missed those.